rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactorySpec.groovy

```diff
@@ -0,0 +1,52 @@
+package com.netflix.spinnaker.rosco.providers.util
+
+import spock.lang.Shared
+import spock.lang.Specification
+
+class LocalJobFriendlyPackerCommandFactorySpec extends Specification {
+
+  @Shared
+  LocalJobFriendlyPackerCommandFactory packerCommandFactory = new LocalJobFriendlyPackerCommandFactory()
+
+  void "baseCommand is included"() {
+    setup:
+      def parameterMap = [
+          something:   "fast"
+      ]
+      String templateFile = "aws-chroot.json"
+
+    when:
+      def packerCommand = packerCommandFactory.buildPackerCommand("sudo", parameterMap, templateFile)
+
```

Comment:
templateFile isnt considered when building the command, right?
So it seems you could have one function that takes a base command argument and expected prefix (list). Then your 3 tests call this function with ("sudo", ["sudo"]), (null, []) and ("", []). Not only is it simpler and clearer, it emphasizes that only the baseCommand is relevant, not the templateFile.

I havent written spock tests in a while so dont know if this makes sense or a good idea. @duftler may disagree.